assets/admin/package.json

```diff
@@ -38,23 +38,24 @@
         "@babel/preset-react": "^7.0.0",
         "@ckeditor/ckeditor5-dev-utils": "^23.5.1",
         "@ckeditor/ckeditor5-theme-lark": "^23.0.0",
-        "autoprefixer": "^9.6.1",
+        "autoprefixer": "^9.8.6",
         "babel-loader": "^8.0.6",
         "clean-webpack-plugin": "^3.0.0",
-        "css-loader": "^3.2.0",
+        "css-loader": "^5.2.4",
         "file-loader": "^6.0.0",
         "glob": "^7.1.2",
-        "mini-css-extract-plugin": "^0.9.0",
+        "mini-css-extract-plugin": "^1.5.0",
         "optimize-css-assets-webpack-plugin": "^5.0.3",
-        "postcss-calc": "^7.0.1",
+        "postcss": "7.0.35",
+        "postcss-calc": "^7.0.5",
         "postcss-hexrgba": "^2.0.0",
-        "postcss-import": "^12.0.1",
+        "postcss-import": "^2.0.1",
```

Comment:
i think this is an error. think it should still be `^12.0.1`

Reply:
fixed!